libnmstate/nm/route.py

```diff
@@ -141,3 +146,45 @@ def _get_default_route_config(gateway, metric, default_table_id, iface_name):
         Route.NEXT_HOP_ADDRESS: gateway,
         Route.METRIC: metric,
     }
+
+
+def add_routes(setting_ip, routes):
+    default_gateway_defined = False
+    for route in routes:
+        if route[Route.DESTINATION] in (IPV4_DEFAULT_GATEWAY_DESTINATION,
+                                        IPV6_DEFAULT_GATEWAY_DESTINATION):
+            if default_gateway_defined:
+                raise NmstateNotImplementedError(
+                    'Only a single default gateway is supported due to '
```

Comment:
This is handling a single profile, isn't it? Not sure what you are getting at...

Reply:
So you are saying that this information should be decoded from the trace and not from the message?
I think it is confusing, but I leave it for the composer discretion, the commit message is good enough.

Reply:
> So you are saying that this information should be decoded from the trace and not from the message?
> I think it is confusing, but I leave it for the composer discretion, the commit message is good enough.

I am still not sure what you are suggesting... Are you proposing to update the message to mention `Only a single default gateway per interface and IP family is supported...`? It could also mention the Bugzilla entry to be filed to allow to specify gateways like normal routes.

Reply:
I am proposing something like: `Only a single default gateway is supported per interface due to...`
BZ/s are mentioned in the commit message, logs do not include such info.